tests/src/Discovery/PathFinderCoreTest.php

```diff
@@ -41,8 +41,26 @@ public function test_find() {
    */
   public function test_getParentDirectory__noParent() {
```

Comment:
OK, the issue is that test_find__noContrib was throwing the exception, but in PathCoreFinder. Editing existing test instead with a mock, so we are back to 100% coverage.